src/components/categories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <h1>Categories</h1>
+    <button type="button">Check Status</button>
+  </div>
+);
+
+export default Categories;
```

Comment:
Nice job creating a React button to display the Under Construction message! Although, as requested by the previous reviewer, please make sure to dispatch the Redux action when clicking on this button 😉. Currently, your button isn't attached to anything, it is just a simple empty button. To make this work, you can attach the `useDispatch` hook to your button and assign the returned message to your front end 😄!

You could have something like this:
Clicking on the button -> Dispatching Redux action -> Save the returned value on a variable -> Display the value on the front end

You are almost there! Cheers and keep coding.